src/test/java/org/sonarsource/sonarlint/ls/mediumtests/ConnectedModeMediumTests.java

```diff
@@ -768,16 +768,6 @@ void shouldThrowExceptionForUnknownProjectKey() {
     awaitUntilAsserted(() -> assertThat(future).isCompletedExceptionally());
   }
 
-  @Test
```

Comment:
This test does not make much sense. Maybe it did when `open in IDE` was not being handled by SLCORE. Now it's a flaky remnant.